### ISSUE TITLE
cannot get gql_POST, status code 400

### ISSUE THREAD

**Issue by alfred-exe** [ading2210/poe-api#37]
This started happening to me just now, and it prevents me from using the module.

```WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (1/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (2/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (3/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (4/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (5/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (6/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (7/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (8/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (9/10)...
WARNING:root:Server returned a status code of 400 while downloading https://poe.com/api/gql_POST. Retrying (10/10)...
Traceback (most recent call last):
  File "C:\Users\alfred\e\Python\server\main 3.11.py", line 54, in <module>
    ai = poe.Client(token)
  File "C:\Users\alfred\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 79, in __init__
    self.subscribe()
  File "C:\Users\alfred\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 160, in subscribe
    result = self.send_query("SubscriptionsMutation", {
  File "C:\Users\alfred\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 147, in send_query
    r = request_with_retries(self.session.post, self.gql_url, json=payload, headers=self.gql_headers)
  File "C:\Users\alfred\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 37, in request_with_retries
      raise RuntimeError(f"Failed to download {url} too many times.")
RuntimeError: Failed to download https://poe.com/api/gql_POST too many times.
```

I went to the URL that it had specified (https://poe.com/api/gql_POST) and the only thing on the page was `{"error":"Method Not Allowed"}`

**Comment by User1231300** [comment]
same

**Comment by TheLime1** [comment]
same here; changing cookies doesn't solve the issue.
Here is the error code:

```
127.0.0.1 - - [14/Apr/2023 23:20:15] "POST /chatbot HTTP/1.1" 500 -
INFO:werkzeug:127.0.0.1 - - [14/Apr/2023 23:20:15] "POST /chatbot HTTP/1.1" 500 -
Traceback (most recent call last):
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2551, in __call__
    return self.wsgi_app(environ, start_response)
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2531, in wsgi_app
    response = self.handle_exception(e)
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "C:\Users\everp\Documents\GitHub\gptCensorFree\chatbot_api.py", line 29, in chatbot
    client = poe.Client(token)
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\poe.py", line 79, in __init__
    self.subscribe()
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\poe.py", line 160, in subscribe
    result = self.send_query("SubscriptionsMutation", {
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\poe.py", line 147, in send_query
    r = request_with_retries(self.session.post, self.gql_url, json=payload, headers=self.gql_headers)
  File "C:\Users\everp\AppData\Local\Programs\Python\Python310\lib\site-packages\poe.py", line 37, in request_with_retries
    raise RuntimeError(f"Failed to download {url} too many times.")
RuntimeError: Failed to download https://poe.com/api/gql_POST too many times.
```

**Comment by PhantomPainX** [comment]
Same. Working perfectly fine minutes ago and suddenly drops this error :(

**Comment by TheLime1** [comment]
@PhantomPainX 
so i think its a server-related issue, not a code issue; maybe the server at Capacity right now? like Chat-Gpt?

**Comment by PhantomPainX** [comment]
> https://poe.com/api/gql_POST

Yep, i think the same, it is a poe's server side problem

**Comment by csmust** [comment]
I had the same problem.

**Comment by ading2210** [comment]
@csmust PR #39 apparently fixes this issue. Try that guy's fork perhaps?

**Comment by ading2210** [comment]
This has been fixed in 0.2.10.
